<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CQDlgImportOptions</class>
 <widget class="QDialog" name="CQDlgImportOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>246</width>
    <height>286</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Import Options</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QDialogButtonBox" name="qqOk">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>221</width>
     <height>32</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>221</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>Mesh scaling</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QRadioButton" name="qqM">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>201</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>1 unit represents 1 meter</string>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="qqCm">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>201</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>1 unit represents 1 centimeter</string>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="qqMm">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>201</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>1 unit represents 1 millimeter</string>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="qqIn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>201</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>1 unit represents 1 inch</string>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="qq10Cm">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>201</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>1 unit represents 10 centimeters</string>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="qqFt">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>201</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>1 unit represents 1 foot</string>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>221</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Mesh orientation</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QRadioButton" name="qqY">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>201</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Y vector is up</string>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="qqZ">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>201</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Z vector is up</string>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>qqM</tabstop>
  <tabstop>qqFt</tabstop>
  <tabstop>qq10Cm</tabstop>
  <tabstop>qqIn</tabstop>
  <tabstop>qqCm</tabstop>
  <tabstop>qqMm</tabstop>
  <tabstop>qqY</tabstop>
  <tabstop>qqZ</tabstop>
  <tabstop>qqOk</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
